File: solutions/python/bob/3/bob.py
def response(hey_bob):
    hey_bob = hey_bob.strip()
    # isupper() return True if all chars are uppercase, otherwise False (also return False if there's any non alphabet char). All CAPITAL means you yell at him.
    if hey_bob.isupper():
        if hey_bob.endswith("?"):
            return "Calm down, I know what I'm doing!"
        return "Whoa, chill out!"
    elif hey_bob.endswith("?"):
        return "Sure."
    elif hey_bob == "":
        return "Fine. Be that way!"
    return "Whatever."

File: solutions/javascript/bob/3/bob.js
export const hey = (message) => {
  let mes = message.trim()

  let regExp = /[A-Z]+/; // I'm testing if there a capital letter in mes

  if (!mes) return "Fine. Be that way!"

  if (regExp.test(mes) && mes.toUpperCase() === mes){
    return (mes.endsWith("?")) ? "Calm down, I know what I'm doing!" : "Whoa, chill out!"
  }

  return (mes.endsWith("?")) ? "Sure." : "Whatever."

};
